scipy/stats/tests/test_stats.py

```diff
@@ -8766,13 +8775,14 @@ class TestPageTrendTest:
     # results could be checked (to limited precision) against tables in
     # scipy.stats.page_trend_test reference [1]
 
-    np.random.seed(0)
-    data_3_25 = np.random.rand(3, 25)
-    data_10_26 = np.random.rand(10, 26)
+    rng = np.random.default_rng(3113562111)
+    data_3_25 = rng.random((3, 25))
+    rng = np.random.default_rng(3113562111)
```

Comment:
why is rng recreated again? `data_10_26` will have overlapping random numbers with `data_3_25`. The previous code would have those two objects with totally different random numbers.

i.e. shouldn't it just be:
```
    rng = np.random.default_rng(3113562111)
    data_3_25 = rng.random((3, 25))
    data_10_26 = rng.random((10, 26))
```

Reply:
Because I accidentally generated the reference values in R using these values and didn't want to do it again.. it requires a lot of manual copy-paying. There is no effect on the legitimacy of the tests.